packages/buendia-utils/data/usr/bin/buendia-init

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+LOG=/var/log/buendia/buendia-init.log
+HWCLOCK_FILE=/var/log/buendia/hwclock.final
```

Comment:
What does the `.final` suffix signify here?

Reply:
That this file was written during shutdown.  (In systemd this target is called `final`.)  Maybe it's cryptic, though; do you think it should be named `hwclock.shutdown` or `hwclock.last` or something better?  Happy to rename.